sympy/solvers/ode.py

```diff
@@ -1202,8 +1203,25 @@ class in it.  Note that a hint may do this anyway if
                             not check.has(zoo) and not check.has(-oo):
                             coeff_dict = {'p': p, 'q': q, 'x0': point, 'terms': terms}
                             matching_hints["2nd_power_series_regular"] = coeff_dict
-
-
+                            #If the ODE has regular singular point at x0 and is of the form
+                            #Eq(x**2*Derivative(y(x),x,x)+x*Derivative(y(x),x) thus Bessel's equation
+                            if p==1:
+                                b4 = Wild('b4', exclude=[x,f(x),df])
```

Comment:
Yes b4 can be anything. No, please elaborate what sort of equations should the Bessel's equation support?